src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+
+    public Lottery() {
+    }
```

Comment:
Do you really need this empty constructor?

Reply:
I was fixed it, thank you